service/src/test/java/uk/gov/hmcts/reform/fpl/utils/OrderIssuedNotificationTestHelper.java

```diff
@@ -18,37 +19,38 @@
 import java.util.List;
 import java.util.Locale;
 import java.util.Map;
+import java.util.function.Supplier;
 
 import static java.nio.charset.StandardCharsets.ISO_8859_1;
-import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.GeneratedOrderType.BLANK_ORDER;
 import static uk.gov.hmcts.reform.fpl.enums.IssuedOrderType.NOTICE_OF_PLACEMENT_ORDER;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.EMAIL;
 import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.POST;
 import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
 import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.formatCaseUrl;
 
-public class NotifyAdminOrderIssuedTestHelper {
+public class OrderIssuedNotificationTestHelper {
 
-    private NotifyAdminOrderIssuedTestHelper() {
+    private OrderIssuedNotificationTestHelper() {
     }
 
     private static final byte[] PDF = {1, 2, 3, 4, 5};
     private static final String EXAMPLE_COURT = "Family Court";
     private static final String callout = "^Jones, SACCCCCCCC5676576567, hearing " + LocalDateTime.now().plusMonths(3)
         .toLocalDate().format(DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM).localizedBy(Locale.UK));
 
-    public static MapDifference<String, Object> verifyNotificationSentToAdminWhenOrderIssued(
-        ArgumentCaptor<Map<String, Object>> dataCaptor, IssuedOrderType issuedOrderType) {
+    public static MapDifference<String, Object> verifyNotification(ArgumentCaptor<Map<String, Object>> captor,
```

Comment:
`verifyNotification` isn't really the best name, the method is only verifying the `jsonObjectFile` and getting a comparison of the rest of the maps

Reply:
moved the assertion into this method and kept name as verifyNotification (which now makes sense)